src/TConsole.h

```diff
@@ -287,6 +287,7 @@ public slots:
 
 private:
     ConsoleType mType;
+    QSize oldSize;
```

Comment:
Can you indicate it is a member - so it shows up as such outside of an IDE - obviously you'll need to change the other places as well:
```suggestion
    QSize mOldSize;
```